components/automate-backend-elasticsidecar/habitat/plan.sh

```diff
@@ -0,0 +1,53 @@
+# shellcheck disable=SC2148
+pkg_name="automate-backend-elasticsidecar"
+pkg_description="Password manager for elasticsearch-odfe"
+pkg_origin="chef"
+vendor_origin="chef"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=("Chef-MLSA")
+pkg_version="1.0.28"
+pkg_upstream_url="http://github.com/chef/a2-ha-backend/components/automate-backend-elasticsidecar"
+pkg_deps=(
+  core/bash
+  core/ruby26
```

Comment:
Is there a reason we're building tools on a version of Ruby that's going to be EOL at the end of the year?

Reply:
We Will be upgrading this in subsequent sprints https://app.zenhub.com/workspaces/the-delta-quadrant-5fc7c4c3922dc10021323fa6/issues/chef/automate/5546